Add rctime object for P2

phonoclese opened the pull request:
This object updates the rctime object from the P1 library for the P2 processor.  

It uses a Smart pin configured with Schmitt trigger to count pin high states and generate an interrupt signal which launches an isr to save the clock count and then recharge the rc circuit.

The code remains resident in cog RAM and runs in the background using a small number of registers beginning at $1B0.

PropGit commented:
Thank you for your contribution; we really appreciate it!

After merging, I noticed the object is in the Human Input folder.  

While it makes sense that it's a Human Input category, it's also in the Sensor category (objects may be in multiple categories).  

To prevent more duplication, we put all new object in the All folder (in a subfolder of their own) and for the category folders, they only contain a README.md that lists and links the objects that belong to that category.

As of commit [b1b508c](https://github.com/parallaxinc/propeller/commit/b1b508c277b15a3836bf99295fb2818e8ea209ea) I've moved your object to the All/p2_rctime folder, and updated the READMEs in the Sensor and Human Input folders to list and link to it.

Make sure to fetch and merge these changes into your local repo for future use.

Thank you!

phonoclese commented:
Thank you for the information.  I am new to Git and wasn’t sure exactly how it was going to work.  In the future I’ll put stuff in the All folder!I updated my fork and see the code.  It is great to be able to offer something up while learning about the P2.  I look forward to seeing new hardware with this incredible processor.PhonosOn Jul 20, 2023, at 9:56 AM, Parallax Git Administrator ***@***.***> wrote:﻿
Thank you for your contribution; we really appreciate it!
After merging, I noticed the object is in the Human Input folder.
While it makes sense that it's a Human Input category, it's also in the Sensor category (objects may be in multiple categories).
To prevent more duplication, we put all new object in the All folder (in a subfolder of their own) and for the category folders, they only contain a README.md that lists and links the objects that belong to that category.
As of commit b1b508c I've moved your object to the All/p2_rctime folder, and updated the READMEs in the Sensor and Human Input folders to list and link to it.
Make sure to fetch and merge these changes into your local repo for future use.
Thank you!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
